test/unit/composite-view/composite-view.spec.js

```diff
@@ -666,9 +659,7 @@ describe('composite view', function() {
         tagName: 'table',
         template: this.gridTemplateFn,
         childView: this.GridRow,
-        attachHtml: function(collectionView, itemView) {
```

Comment:
I don't know why these tests were written this way.. super odd... and not how `attachHtml` works in `NextCollectionView`